brainstormforce/ultimate-addons-for-gutenberg
# Improvement - Load static CSS assets from custom stylesheet when file generation is disabled

### Description

- Load assets from custom stylesheet when file generation is disabled

### Types of changes




- Improvement

### How has this been tested?

- Activate block from backend and check is the CSS for the activated block is generated in `custom-style-css.css`?
- e.q Activate Advanced Heading block > Drag and drop on page > Add a style check frontend. everything looks good.
- Check static CSS for Advanced heading is loaded from `custom-style-blocks.css` or not.
- And `uagb-style-frontend-xxx` only contain dynamic css.

### Checklist:
